components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -121,6 +122,11 @@ data class Addon(
      */
     fun isSupported() = installedState?.supported == true
 
+    /**
+     * Returns whether or not this [Addon] is currently disabled because it is not supported.
+     */
+    fun isDisabledAsUnsupported() = installedState?.disabledAsUnsupported == true
```

Comment:
If `isDisabledAsUnsupported` is true should `supported` be `false`?

Reply:
@Amejia481 I've added a comment. Both flags can be `true` at the same time. We might disable an extension during migration. It's state is then disabled as unsupported. Later this extension might be come supported as indicated by our AMO response. The both flags are true. The extension is now supported, but it is still disabled as unsupported. This is important because we can then trigger a flow to automatically enable those extensions. (We still need to discuss how this will look like to the user, see mocks.)

Reply:
👍